dimod/io/coo.py

```diff
@@ -0,0 +1,67 @@
+import re
+
+from dimod.binary_quadratic_model import BinaryQuadraticModel
+
+
+def dumps(bqm):
+    """Dump a binary quadratic model to a string in COOrdinate format."""
+    return '\n'.join(_iter_triplets(bqm))
+
+
+def dump(bqm, fp):
+    """Dump a binary quadratic model to a string in COOrdinate format."""
+    for triplet in _iter_triplets(bqm):
+        fp.write('%s\n' % triplet)
+
+
+def loads(s, bqm):
+    """Load a COOrdinate formatted binary quadratic model from a string."""
+    pattern = re.compile(r'^(\d+)[ ]+(\d)+[ ]+([+-]?([0-9]*[.])?[0-9]+)')
```

Comment:
Few comments:
- second number is not captured properly if it has multiple-digits (`1 22 3`)
- third number (floating point) can have leading zeros (`0001.2`); scientific notation support?
- to support other whitespace characters (like tab), you could use `\s` instead of `[ ]`.
- `$` anchor is missing
- you might consider stripping whitespace, or adding `\s*` at the beginning and end of regex